src/careamics/config/support/supported_data.py

```diff
@@ -60,9 +60,9 @@ def _missing_(cls, value: object) -> str:
         return super()._missing_(value)
 
     @classmethod
-    def get_extension(cls, data_type: Union[str, SupportedData]) -> str:
+    def get_extension_pattern(cls, data_type: Union[str | SupportedData]) -> str:
```

Comment:
`Union[A | B]` is probably a typo?

Since we are still supporting py9, so we should avoid using pipes for Unions.

Reply:
Oh yeah oops, not sure how this happened